### DISCUSSION TITLE
I am modifying the dashboard design

### DISCUSSION THREAD

**Discussion by virla01** [filamentphp/filament#14774] [Help]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

I am modifying the dashboard design and I have a problem with both:
filament()->isGlobalSearchEnabled() and filament()->hasDatabaseNotifications() which give me false, how can I solve this? I say that I moved it for design reasons

**Comment by leandrocfe** [comment]
Please explain more about it..

**Reply by virla01** [reply]
I use the topbar but in another place not along with the logo, the global search, the notifications and the user menu, that is why I have to put the global search and the notifications in the main bar with the logo, I hope it is understood, in the code, the top menu is positioned in the main bar, and that is my problem, actually not so much, since there is an if that asks the question that I put in my post, and I do not want to delete it
